app/bower_components/angular-cryptography/README.md

```
@@ -0,0 +1,74 @@
+[Middleout Angular Cryptography](http://ngmodules.org/modules/angular-cryptography)
+==================
+
+AngularJS Module that integrate cryptography functionality offers from the [crypto-js](https://code.google.com/p/crypto-js/) project. Provides a simple service to encrypt and decrypt using AES.
+
+Dependencies
+------------
+- [AngularJS 1.1.4 + ](http://angularjs.org/) (tested with 1.1.4 and 1.2.16)
```

markdownlint-cli2 (0.17.2)

Spaces inside link text
null

(MD039, no-space-in-links)